BusinessLayer/InventoryCsvManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Models;
+using Serilog;
+
+namespace BusinessLayer
+{
+    public class InventoryCsvManager : IInventoryManager
```

Comment:
is inventory the best name to describe what you are doing?

Reply:
Inventory seems like a good name.
We cannot name it LogManager, so any idera is welcomed if Inventory is not explicit enough